src/modules/about-us/OurValues/ourValues.tsx

```diff
@@ -0,0 +1,37 @@
+"use client";
+import Image from "next/image";
+import { ourValuesList } from "./valuesHelper";
+
+type ourValuesProps = {
+  name: string;
+  description: string;
+  imageurl: string;
+};
+
+const OurValues = () => {
+  return (
+    <>
+      <section className="flex flex-col justify-center gap-16 md:flex-col lg:mx-48">
+        <h2 className="text-center font-bold text-gray-600">Our Values</h2>
+        <section className="grid grid-cols-1 justify-between sm:gap-16 lg:grid-cols-3 lg:gap-12">
+          {ourValuesList.map(
+            ({ name, description, imageurl }: ourValuesProps) => (
```

Comment:
Shouldn't require `: ourValuesProps` here, since we are using TS, `ourValuesList` should have type defined already